src/pages/technologies/react.md

```diff
@@ -138,9 +147,9 @@ Screenshots
 
 - [Jack Herrington - Mastering React Context: Do you NEED a state manager?](https://www.youtube.com/watch?v=MpdFj8MEuJA)
 
-### React Memo
+### React Portals
 
-- [Jack Herrington - Mastering React Memo](https://www.youtube.com/watch?v=DEPwA3mv_R8)
+- [Theo - ping.gg - This React Feature BREAKS THE RULES??? 😱 (how to use React Portals and why?)](https://www.youtube.com/watch?v=IgJcK0SbiLM)
```

Comment:
I'd suggest removing clickbait part from the title

Reply:
I think you're right. This does not add anything valuable to the title.